polkadot/xcm/src/lib.rs

```diff
@@ -339,13 +514,23 @@ impl<RuntimeCall> From<v3::Xcm<RuntimeCall>> for VersionedXcm<RuntimeCall> {
 	}
 }
 
+impl<RuntimeCall> From<v4::Xcm<RuntimeCall>> for VersionedXcm<RuntimeCall> {
```

Comment:
Curious why you have to manually write this; does the macro not auto-generate this impl?

Reply:
Not this impl. I guess it's because of the generic?